### DISCUSSION TITLE
Call fetch hook on client side on SSR app

### DISCUSSION THREAD

**Discussion by adarsh4d** [nuxt/nuxt#9518] [Questions (Nuxt 2)] [answered]
So I am using `@nuxt/firebase` since it doesn't support server auth calls. I cannot use the fetch hook on authenticated routes on the server. 

So I have disabled fetch on the server side by `fetchOnServer: false`,  Is it possible to call the same fetch hook again on the client-side once the page hits the browser?

**Comment by danielroe** [comment, marked as answer]
If you set `fetchOnServer: false`, then it will run on the client instead automatically. You can also trigger a fetch at any point with `this.$fetch()` within the component.

(A note: you should also make sure that your fetch function doesn't receive a context argument, as that changes its behaviour.)

**Reply by adarsh4d** [reply]
Thanks for the quick reply, it works now!! I was using the store object via the context, removing that fixed the issue. It would be helpful if this is added to the docs.
